README.md

```diff
@@ -62,7 +62,7 @@ To build the Riak Java Client, you must have the correct version of the riak_pb
 
 ```
 git clone https://github.com/basho/riak_pb
-git checkout java-2.1.1.0
+git checkout 2.1.2.0
```

Comment:
We include riak_pb as a submodule now, so we can remove this step once that develop is merged back to this branch.